sdk/lusid/utilities/api_configuration.py

```diff
@@ -4,7 +4,7 @@
 class ApiConfiguration:
 
     def __init__(self, token_url=None, api_url=None, username=None, password=None, client_id=None, client_secret=None,
-                 app_name=None, certificate_filename=None, proxy_config=None):
+                 app_name=None, certificate_filename=None, proxy_config=None, access_token=None):
```

Comment:
Here we are adding the **access_token** as an attribute on the ApiConfiguration. 
This attribute gets loaded from the env vars.